vignettes/EpiNow2.Rmd

```diff
@@ -293,3 +296,137 @@ estimates$summary$summary_plot
 ```
 
 ![plot of chunk plot_regional_epinow_summary](EpiNow2-plot_regional_epinow_summary-1.png)
+
+#### When regions have different settings
+
+Region-specific settings can be applied with the [`opts_list()`](https://epiforecasts.io/EpiNow2/reference/opts_list.html) function.
+
+As an example, let's use `reported_cases` from the previous example and define a 7-day random walk for testland and maintain the default for realland.
+
+
+``` r
+rt <- opts_list(
+  rt_opts(),
```

Comment:
this now uses a different intercept prior

Reply:
Are you requesting that I revert to the previous prior? Not sure I get the comment.

Reply:
I've changed to using ascertainment in f11f846e3808726bb9b92e97a78449f85f7294af